generator.rb

```diff
@@ -0,0 +1,85 @@
+def is_vowel?(word)
+    word.each_char do |char|
```

Comment:
When you use the `is_vowel?` method, you are using it while you're inside of an `each_char`, and you're passing into `is_vowel?` a single character.

I think that `word` is not an accurate name for this variable, and I think that you don't need to do `each_char` for this variable